app/code/core/Mage/Adminhtml/controllers/Permissions/UserController.php

```diff
@@ -111,7 +111,7 @@ public function saveAction()
             $id = $this->getRequest()->getParam('user_id');
             $model = Mage::getModel('admin/user')->load($id);
             // @var $isNew flag for detecting new admin user creation.
-            $isNew = !$model->getId() ? true : false;
+            $isNew = $model->getId() ? false : true;
```

Comment:
Is OK?
```suggestion
            $isNew = !$model->getId() ;
```

Reply:
@kiatng apllied according rule ... only difference, rector casts it to `bool`, that looks more accurate.